File: src/main/java/com/bitvote/project/coin/Coin.java
package com.bitvote.project.coin;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Coin {
    private String id;
    private String symbol;
    private String name;
    private String image;
    private String current_price;
    private String market_cap;
    private String market_cap_rank;
    private String high_24h;
    private String low_24h;
    private String price_change_percentage_24h;
    private String market_cap_change_percentage_24h;
    private String market_cap_change_24h;
}

File: src/model/coin.ts
export interface Coin {
    id: number;
    symbol: string;
    name: string;
    image: string;
    current_price: number;
    market_cap: number;
    market_cap_rank: number;
    high_24h: number;
    low_24h: number;
    price_change_percentage_24h: number;
    market_cap_change_percentage_24h: number;
    market_cap_change_24h: number;
    total_volume: number;
}